tests/Controller/JsonApiTestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Sylius\Tests\Controller;
+
+class JsonApiTestCase extends \Lakion\ApiTestCase\JsonApiTestCase
```

Comment:
`Lakion\ApiTestCase\JsonApiTestCase` should be imported under alias e.g. `LakionJsonApiTestCase` or `BaseJsonApiTestCase`